plugins/src/views/trans.json

```diff
@@ -83,6 +83,26 @@
       "act_create_normal": "創建5V5單排房間",
       "act_practice_tool": "創建練習模式",
       "act_quit_pvp": "跳GAME(僅限PVP模式)"
+    },
+        {
+      "_locales": [
+        "tr-TR"
+      ],
+      "welcome_msg": "Merhaba sihirdar! Pengu'n başarılı bir şekilde çalıştı! Umarım iyi bir zaman geçirirsin. Topluluğumuza katılarak 😎 temalara ve eklentilere ulaşabilirsin! 👇",
```

Comment:
instead of 
> Topluluğumuza katılarak 😎 temalara ve eklentilere ulaşabilirsin!

change it to
> Topluluğumuza katılarak 😎 harika temalara ve eklentilere ulaşabilirsin!

this makes it feel engaging and natural.